spec/ref_arch_setup/install_spec.rb

```diff
@@ -47,4 +47,76 @@
       expect(install.bootstrap_mono(pe_conf_path, pe_tarball_path, target_master)).to eq(false)
     end
   end
+
+  describe "make_tmp_work_dir" do
+    it "with defaults and got a pass from bolt_helper" do
```

Comment:
It seems like it would be more clear to use contexts to separate the state from the expected result. But let me know if that seems like overkill in this case :)